src-tauri/src/lib.rs

```diff
@@ -314,6 +319,97 @@ pub fn launch_northstar(
     ))
 }
 
+pub fn get_titanfall_proton() -> Option<String> {
+    let steampath = SteamDir::locate().unwrap();
+    let configpath = steampath.path.join("config").join("config.vdf");
+
+    let vdf_text = fs::read_to_string(configpath).unwrap();
+    let config = Vdf::parse(&vdf_text).unwrap().value;
+    let obj = config.unwrap_obj();
+
+    let software = obj.get("Software")?.get(0)?.get_obj()?;
+    let valve = software.get("Valve")?.get(0)?.get_obj()?;
+    let steam = valve.get("Steam")?.get(0)?.get_obj()?;
+    let compat_tool_mapping = steam.get("CompatToolMapping")?.get(0)?.get_obj()?;
+    let tfapp = compat_tool_mapping
+        .get(TITANFALL2_STEAM_ID)?
+        .get(0)?
+        .get_obj()?;
+    let name = tfapp.get("name")?.get(0)?.get_str()?.to_string();
```

Comment:
This whole parsing mess is cause the steam vdf parsing crate we use doesn't support parsing for proton version, right?

Reply:
Yes.

I can go and submit a PR to `steamlocate-rs` to add a helper function to fetch the compatibility tool for a given appid if that desired.

Reply:
I mean, I'm not gonna stop you if you want to ;)

Reply:
https://github.com/WilliamVenner/steamlocate-rs/pull/18

Reply:
I don't think the steamlocate-rs PR will be merged any time soon.

Reply:
Yeah, I guess we'll just have to keep the messy part in FlightCore then ^^"

Reply:
Should the PR get merged we can always strip it out again.